aries-site/src/data/structures/components.js

```diff
@@ -362,6 +362,7 @@ export const components = [
   },
   {
     name: 'FileInput',
+    accessibility: 'Passed WCAG 2.2 AA',
```

Comment:
I think we are not meeting Wcag level A
```suggestion
    accessibility: 'Failed WCAG 2.2 A',
```

Reply:
Accepted change